src/components/App.js

```diff
@@ -19,11 +20,18 @@ let mapStateToProps = (state) => {
 class App extends React.Component {
   constructor(props) {
     super(props)
+    this.state = {
+      display : "Board"
```

Comment:
instead of using a state variable with a string to represent what is being shown, it would be easier if you had them as booleans, For example, a variable named isBoardVisible or isRulesVisible would help and you will just have to check for them, instead of string comparisons.

Reply:
Okay. I used strings so that the code becomes readable.
Thanks for the suggestion 
@cheevu-RS